src/config/configJsonSchema.js

```diff
@@ -36,13 +39,47 @@ module.exports = {
           type: 'object',
           properties: {
             port: {
-              type: 'integer',
-              minimum: 0,
+              $ref: '#/definitions/port',
+            },
+            seeds: {
+              type: 'array',
+              items: {
+                type: 'object',
+                properties: {
+                  host: {
+                    type: 'string',
```

Comment:
maxLength: 1

Reply:
Why? The host string is usually more than 1 character.

Reply:
Ops.. I meant minLength :)